LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/resize-observer/notify-expected.txt

```diff
@@ -0,0 +1,23 @@
+ResizeObserver tests
+
+t1
+t2
+t3
+inline
+
+PASS ResizeObserver implemented
+PASS guard
+PASS test0: notification ordering
+PASS test1: display:none triggers notification
+PASS test2: remove/appendChild trigger notification
+PASS test3: dimensions match
+PASS test4: transform do not cause notifications
+PASS test5: moving an element does not trigger notifications
+FAIL test6: inline element notifies once with 0x0. assert_unreached: Timed out waiting for notification. (1000ms) Reached unreachable code
```

Comment:
Why is this failing in WK1?

Reply:
updaterendering is not triggered properly in WK1, filed a bug https://bugs.webkit.org/show_bug.cgi?id=250900

Reply:
Please link to that bug in the commit message.

Reply:
Done!